files/materialized_view_works_workgenres.sql

```diff
@@ -40,7 +40,7 @@ as
      JOIN licensepools ON editions.id = licensepools.presentation_edition_id
      JOIN datasources ON licensepools.data_source_id = datasources.id
      JOIN identifiers on licensepools.identifier_id = identifiers.id
-     LEFT JOIN workgenres ON works.id = workgenres.work_id
+     JOIN workgenres ON works.id = workgenres.work_id
```

Comment:
This is changing the SQL back to what it used to be. Rather than modifying an existing materialized view, this branch creates a new one.